HealthVault/Classes/Clients/MHVPersonClient.m

```diff
@@ -70,56 +130,274 @@ - (void)getAuthorizedPeopleWithCompletion:(void(^)(NSArray<MHVPersonInfo *> *_Nu
     {
         return;
     }
+
+    [self getAuthorizedPeopleWithSettings:[MHVGetAuthorizedPeopleSettings new]
+                              personInfos:nil
+                               completion:completion];
+}
+
+- (void)getAuthorizedPeopleWithSettings:(MHVGetAuthorizedPeopleSettings *)settings
+                             completion:(void(^)(NSArray<MHVPersonInfo *> *_Nullable personInfos, NSError *_Nullable error))completion
+{
+    MHVASSERT_PARAMETER(settings);
+    MHVASSERT_PARAMETER(completion);
+    
+    if (!completion)
+    {
+        return;
+    }
+
+    if (!settings)
+    {
+        completion(nil, [NSError MVHRequiredParameterIsNil]);
+        return;
+    }
+
+    [self getAuthorizedPeopleWithSettings:settings
+                              personInfos:nil
+                               completion:completion];
+}
+
+- (void)getAuthorizedPeopleWithSettings:(MHVGetAuthorizedPeopleSettings *)settings
+                            personInfos:(NSArray<MHVPersonInfo *> *_Nullable)personInfos
+                             completion:(void(^)(NSArray<MHVPersonInfo *> *_Nullable personInfos, NSError *_Nullable error))completion
```

Comment:
It looks like there is a MHVPersonInfoCollection - Maybe we should use that instead.